components/eam/bld/configure

```diff
@@ -1062,25 +1062,14 @@ if ($print>=2) { print "Using $pbl_pkg PBL scheme.$eol"; }
 
 # Set default
 my $rad_pkg = 'rrtmg';
-if ($phys_pkg =~ m/^ideal$|^adiabatic$|^cam[34]$/) {
-    $rad_pkg = 'camrt';
-}
```

Comment:
Can you put this line back and change it to RRTMG? 
I'll need this when I revive the idealized physics compsets.